htmlreport/setup.py

```diff
@@ -1,21 +1,31 @@
 #!/usr/bin/env python3
+import shutil
+from setuptools import find_packages, setup
 
-from setuptools import setup
-
-with open('README.txt') as f:
+with open("README.txt") as f:
     readme = f.read()
 
+shutil.copy("cppcheck-htmlreport", "cppcheck_htmlreport/run.py")
+
 setup(
-    name="cppcheck",
-    description='Python script to parse the XML (version 2) output of ' +
-                'cppcheck and generate an HTML report using Pygments for syntax ' +
-                'highlighting.',
+    name="cppcheck-htmlreport",
+    description=(
+        "Python script to parse the XML (version 2) output of "
+        "cppcheck and generate an HTML report using Pygments for syntax "
+        "highlighting."
+    ),
     long_description=readme,
-    author='Henrik Nilsson',
-    url='https://github.com/danmar/cppcheck',
-    license='GPL',
-    scripts=[
-        "cppcheck-htmlreport",
-    ],
-    install_requires=['Pygments']
+    author="Henrik Nilsson",
```

Comment:
@danmar Erm...

Reply:
yes.. could we change author to "Cppcheck team" perhaps?

Reply:
Updated that.